modules/phase_field/tests/MultiPhase/tests

```diff
@@ -3,11 +3,31 @@
     type = 'Exodiff'
     input = 'derivativetwophasematerial.i'
     exodiff = 'derivativetwophasematerial_out.e'
-    max_parallel = 1                              # -pc_type lu
+    max_parallel = 1
```

Comment:
Pushed a new commit to remove the lu. The test runs faster without it anyways (2x).